### ISSUE TITLE
Unable to run project files to check accessibility violations using axe-core/cli

### ISSUE THREAD

**Issue by jeevanm18** [dequelabs/axe-core-npm#486]
Thanks for reporting an issue to axe-core-npm. Please provide all necessary info to reproduce the issue. Without adequate details, your issue may be closed without investigation.

If you’re reporting a bug, include a description of the issue and a page or code snippet where it can be reproduced. Please make sure you have tested this with the latest version of `@axe-core/*`.

Describe what the desired behavior would be.
Im trying to run the command  view % `axe https://{project url}  --timeout=120 --save results.json --tags wcag2a --verbose --browser chrome --show-errors`

Product: axe-core/cli

Expectation: Want this command to execute all the files in the project and report accessibility violations

Actual:  It opens the browser within few millisecond browser gets close and the error An error occurred while testing this page.is displayed in terminal without   --show-errors command  with --show--errors  below error is seen 

```
TypeError: Failed to set the 'innerHTML' property on 'Element': This document requires 'TrustedHTML' assignment.
    at eval (eval at executeAsyncScript (:552:26), <anonymous>:6:28)
    at eval (eval at executeAsyncScript (:552:26), <anonymous>:9:10)
    at executeAsyncScript (<anonymous>:552:47)
    at <anonymous>:567:29
    at callFunction (<anonymous>:446:22)
    at <anonymous>:460:23
    at <anonymous>:461:3
  (Session info: chrome=99.0.4844.83)
    at Object.throwDecodedError (/Users/jm/.nvm/versions/node/v14.13.1/lib/node_modules/@axe-core/cli/node_modules/selenium-webdriver/lib/error.js:522:15)
    at parseHttpResponse (/Users/jm/.nvm/versions/node/v14.13.1/lib/node_modules/@axe-core/cli/node_modules/selenium-webdriver/lib/http.js:548:13)
    at Executor.execute (/Users/jm/.nvm/versions/node/v14.13.1/lib/node_modules/@axe-core/cli/node_modules/selenium-webdriver/lib/http.js:474:28)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async Driver.execute (/Users/jm/.nvm/versions/node/v14.13.1/lib/node_modules/@axe-core/cli/node_modules/selenium-webdriver/lib/webdriver.js:735:17) {
  remoteStacktrace: '0   chromedriver                        0x0000000103e82159 chromedriver + 5120345\n' +
    '1   chromedriver                        0x0000000103e0fb13 chromedriver + 4651795\n' +
    '2   chromedriver                        0x00000001039ffe68 chromedriver + 392808\n' +
    '3   chromedriver                        0x0000000103a03619 chromedriver + 407065\n' +
    '4   chromedriver                        0x0000000103a039cb chromedriver + 408011\n' +
    '5   chromedriver                        0x0000000103a6674b chromedriver + 812875\n' +
    '6   chromedriver                        0x0000000103a52b42 chromedriver + 731970\n' +
    '7   chromedriver                        0x0000000103a65637 chromedriver + 808503\n' +
    '8   chromedriver                        0x0000000103a52a33 chromedriver + 731699\n' +
    '9   chromedriver                        0x0000000103a285dd chromedriver + 558557\n' +
    '10  chromedriver                        0x0000000103a294f5 chromedriver + 562421\n' +
    '11  chromedriver                        0x0000000103e3f38d chromedriver + 4846477\n' +
    '12  chromedriver                        0x0000000103e5921c chromedriver + 4952604\n' +
    '13  chromedriver                        0x0000000103e5ea12 chromedriver + 4975122\n' +
    '14  chromedriver                        0x0000000103e59b4a chromedriver + 4954954\n' +
    '15  chromedriver                        0x0000000103e345b0 chromedriver + 4801968\n' +
    '16  chromedriver                        0x0000000103e73f78 chromedriver + 5062520\n' +
    '17  chromedriver                        0x0000000103e740ff chromedriver + 5062911\n' +
    '18  chromedriver                        0x0000000103e89545 chromedriver + 5150021\n' +
    '19  libsystem_pthread.dylib             0x00007fff2038c8fc _pthread_start + 224\n' +
    '20  libsystem_pthread.dylib             0x00007fff20388443 thread_start + 15\n'
```

Motivation: Since the expectation is to run all the files in project

<pre><code>
axe-core version: 4.4.1
@axe-core/[integration]: X.Y.Z

- Node version: 14.13.1
- Platform:  Mac 
</code></pre>
